src/pages/guides/getting_started/local_development/dev_tooling.md

```diff
@@ -24,17 +24,23 @@ This section provides a set of guides to help you in the development stage of yo
 
 ## Add-on Development mode
```

Comment:
This need not be the first section anymore. Perhaps lets move it after https://developer-stage.adobe.com/express/add-ons/docs/guides/getting_started/local_development/dev_tooling/#start-script-options section or combine with the 'Sideload your addon' section?
I say so because we dont want devs to think that dev mode activation is a prestep to add-on development. They will be able to view and accept DTOU, enable dev mode when they get to add-on testing window.

Reply:
We can also include a screenshot of the CLI terminal with the shortcut URL that takes the dev directly to the add-on testing window.